src/settings/TranslatorSettingsDialog.ui

```diff
@@ -17,7 +17,7 @@
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
-      <number>0</number>
+      <number>1</number>
```

Comment:
What's happening here? Is that accidental or?

Reply:
Ok, appears I have changed it... No idea what it does, as the change was made through the GUI.....

Reply:
I think it is the tab that is selected. So this would entail the default tab would be the Languages tab? I've reverted it.